aten/src/ATen/native/sparse/SparseCsrTensor.cpp

```diff
@@ -460,9 +460,9 @@ Tensor empty_like_sparse_csr(
         self.col_indices().clone(),
         at::empty(self.values().sizes(), options.layout(kStrided)),
```

Comment:
I think we need to pass "device" to empty as well? Neither `_sparse_csr_tensor_unsafe` nor `SparseCsrTensorImpl::set_member_tensors` do a conversion to the correct device.

Reply:
No, I just fully missed what "layout" actually does here....